db/schema.rb

```diff
@@ -36,6 +36,7 @@
     t.bigint "merchant_id"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.integer "quantity_purchased", default: 0, null: false
```

Comment:
Was having a strange issue where in development quantity was defaulting to 0 but the test environment was nil